src/D2L.Bmx/UpdateChecker.cs

```diff
@@ -62,7 +62,7 @@ internal static class UpdateChecker {
 			responseStream,
 			SourceGenerationContext.Default.GithubRelease
 		);
-		string version = releaseData?.TagName?.Replace( "v", "" ) ?? string.Empty;
+		string version = releaseData?.TagName?.Trim( 'v' ) ?? string.Empty;
```

Comment:
I think we specifically want `TrimStart()` here IIUC

Reply:
Oh ye I see docs says it trims beginning and end. I switched